packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -2,20 +2,24 @@ import SfOverlay from "../../atoms/SfOverlay/SfOverlay.vue";
 import SfIcon from "../../atoms/SfIcon/SfIcon.vue";
 export default {
   name: "SfModal",
+  model: {
+    prop: "visible",
+    event: "toggle"
```

Comment:
`toggle` is inside action and can just close `SfModal`  line 55

Reply:
`close` is a inside action, `toggle` is a event emitted with a boolean value, not?
To be honest, as the event ships a boolean value, imo `toggle` is a good name (even if used only to close now).
I mean, `this.$emit('close', false)` looks like opening the modal (?), but it's closing, wdyt @przspa @aniamusial ?

Reply:
Actually, we just don't need the `close` event, now we're just emitting event to support `v-model`, right?
Maybe we can call `this.$emit('toggle', false)` directly instead of `this.close()`.

Reply:
`close() `is also used for` checkPersistance `method